extra-libs/libdispatch/autobuild/defines

```diff
@@ -0,0 +1,12 @@
+PKGNAME=libdispatch
+PKGSEC=libs
+PKGDES="A library provides support for concurrency on multicore hardware"
```

Comment:
Support library for concurrent code execution on multi-core hardware.

Adopted from [upstream homepage](https://apple.github.io/swift-corelibs-libdispatch/).